src/wallet/wallet.h

```diff
@@ -1038,6 +1043,9 @@ class CWallet final : public WalletStorage, public interfaces::Chain::Notificati
 
     //! Whether the (external) signer performs R-value signature grinding
     bool CanGrindR() const;
+
+    //! Handling wallet HD Master Keys that are used for new automatic descriptors. No derivation will be performed.
+    bool LoadActiveHDPubKey(const CExtPubKey& xpub, const std::optional<CKey>& key, const std::vector<unsigned char>& crypted_key) EXCLUSIVE_LOCKS_REQUIRED(cs_wallet);
```

Comment:
In commit "wallet: Load activehdkey and its private keys" (2e840bcfc0308a461d210e0c7ed66a9bbc723b62)

What does no deriviation will be performed mean? Would suggest clarifying, or maybe removing.

Reply:
Removed